discovery-infra/test_infra/helper_classes/kube_helpers/secret.py

```diff
@@ -1,17 +1,12 @@
-import logging
-import os
 import json
-
-from typing import Optional
+import logging
 
 from kubernetes.client import ApiClient, CoreV1Api
 from kubernetes.client.rest import ApiException
 
-from tests.conftest import env_variables
-
+from ... import consts
```

Comment:
Avoid relative paths like this

Reply:
Why?

Reply:
Because it's bad practice. It restricts how the scripts could be executed, and it's hard to keep track of directories structure and organization.